crates/dwallet-mpc-types/src/dwallet_mpc.rs

```diff
@@ -105,15 +105,15 @@ pub struct NetworkDecryptionKeyPublicData {
 
     pub state: NetworkDecryptionKeyPublicOutputType,
     /// The public output of the `latest` decryption key update (NetworkDKG/Reshare).
-    pub latest_public_output: MPCPublicOutput,
+    pub latest_public_output: VersionedSecpNetworkDkgOutput,
 
     /// The public parameters of the decryption key shares,
     /// updated only after a successful network DKG or Reshare.
     pub decryption_key_share_public_parameters: Vec<u8>,
 
     /// The public output of the `NetworkDKG` process (the first and only one).
     /// On first instance it will be equal to `latest_public_output`.
-    pub network_dkg_output: MPCPublicOutput,
+    pub network_dkg_output: VersionedSecpNetworkDkgOutput,
```

Comment:
Don't do `secp` specific naming, in the future itll be used to hold more than just secp maybe. 
```suggestion
    pub network_dkg_output: VersionedNetworkDkgOutput,
```